.github/workflows/respec.yaml

```diff
@@ -32,10 +34,10 @@ jobs:
     - name: Install dependencies
       run: npm ci
 
-    - uses: actions/checkout@v4 # checkout gh-pages branch
+    - uses: actions/checkout@v4
       with:
-        ref: gh-pages
-        path: deploy
+        ref: main
+        path: docs/deploy
 
     - name: run main script
       run: scripts/md2html/build.sh
```

Comment:
This script needs to be adjusted because it expects the `deploy` folder in the repo root, not nested in `docs`:
https://github.com/OAI/OpenAPI-Specification/blob/46770e807da6a22102a774aa37e43a422801254d/scripts/md2html/build.sh#L43

And while we are at it, we could get rid of the `deploy` thingy as mentioned above and adjust the script to recreate the files `docs/oas/*.html`

Reply:
Nice catch! Fixed in https://github.com/OAI/OpenAPI-Specification/pull/3868/commits/e7a59992cfd4f34cf4d22eb322102622681bfb0a